tracer/src/Datadog.Trace/Headers/NameValueHeadersCollection.cs

```diff
@@ -25,24 +23,12 @@ public NameValueHeadersCollection(NameValueCollection headers)
             _headers = headers;
         }
 
-        public IEnumerable<string> GetValues(string name)
-        {
-            return _headers.GetValues(name) ?? Enumerable.Empty<string>();
-        }
+        public StringValues GetValues(string name) => _headers.GetValues(name);
```

Comment:
Should this be like the other cases, or is this always non-null?
```suggestion
        public StringValues GetValues(string name) => _headers.GetValues(name) ?? StringValues.Empty;
```

Reply:
Good catch, although in the end the behavior should be the same either way. If `_headers.GetValues()` returns `(string[])null`, is it converted to `StringValues` via the implicit conversion from `string[]`. `StringValues` treats null strings, null arrays, or empty arrays the same way (they all mean "there is no value"). (It also treats single-element arrays and a string as equivalent, "there is one value").